### DISCUSSION TITLE
Add Android 11 Bubbles Support

### DISCUSSION THREAD

**Discussion by fukemy** [invertase/react-native-firebase#6799] [Polls]
***Why it is needed***
With the latest update of Android, Android 11, Bubbles are available for all apps. This is a cool feature and should be implemented so we, react-native developers can use it

***Possible implementation***
As I am really not expert in Java, I have no idea to add it. But I found an article about it and attached it below

Code sample
[Implemention of Bubbles in Android 11](https://www.raywenderlich.com/18503960-bubbles-conversations-in-android-11-getting-started)

**Comment by mikehardy** [comment]
This discussion has no real bearing on react-native-firebase and no further consideration of this feature in this repository will be forthcoming


react-native-firebase is concerned with the reception of Firebase Cloud Messages and that's essentially it, nothing related to local display of anything will be added here. you be interested in Notifee.app
